Classes/Transport/LoggingTransport.php

```diff
@@ -31,7 +33,7 @@ class LoggingTransport implements TransportInterface
 
     /**
      * @Flow\Inject
-     * @var \Neos\Flow\Log\SystemLoggerInterface
+     * @var LoggerInterface
      */
     protected $systemLogger;
```

Comment:
Why not change this to `$logger`? Or use `PsrSystemLoggerInterface`. But like this it seems inconsistent, at least in the light of "what I'm used to."

Reply:
Yep I also renamed it to logger everywhere I could in other code parts.